pkg/cmd/render/certs_test.go

```diff
@@ -133,6 +133,7 @@ func assertBundleCorrectness(t *testing.T, secrets []corev1.Secret, bundles []co
 			return bytes.Compare(bundleCerts[i].Raw, bundleCerts[j].Raw) < 0
 		})
 
+		require.Equal(t, len(signers), len(bundleCerts))
```

Comment:
redundant, just for testing ?

Reply:
yep, I was debugging some issue where the informer-based secret listing would return only one signer secret instead of two - which then created a bundle of different size. 
It's just a more explicit assertion than the unwieldy equals output

Reply:
I have also met some issue where the informer-based secret or configmap return different value, but that is found in the kube-controller-mamager-operator, and I have found the bug in the library-go:
https://github.com/openshift/library-go/pull/1532/files


Hope it can give you some inspiration